app/controllers/TrustAuthController.scala

```diff
@@ -173,4 +175,16 @@ class TrustAuthController @Inject()(cc: ControllerComponents,
           .exists(_.value equals value)
     }
   }
+
+  def authoriseAccessCode(draftId: String): Action[AnyContent] = identifierAction {
+    implicit request =>
+      request.body.asJson match {
+        case Some(JsString(accessCode)) =>
+          val accessCodes = config.accessCodes.map(decode)
+          Ok(Json.toJson(TrustAuthAllowed(accessCodes.contains(accessCode))))
```

Comment:
It is going to be difficult to know if a user has been authorised or not looking in Kibana if all the responses are 200's?
Could we return a 401 Unauthorised if the code isn't valid?